services/skipchain/skipchain.go

```diff
@@ -456,6 +461,20 @@ func (s *Service) lenSkipBlocks() int {
 	return len(s.SkipBlocks)
 }
 
+// saves the actual identity
+func (s *Service) save() {
```

Comment:
It comes back to the discussion we had about what should be done for the services - shall we offer a `save`-method in the `Context`? We'd have two use-cases already!